extension/json/src/functions/table_functions/json_scan.cpp

```diff
@@ -21,6 +23,27 @@ using namespace kuzu::common;
 
 struct JsonScanBindData;
 
+static constexpr idx_t JSON_SCAN_FILE_IDX = 0;
```

Comment:
I would move this filed under JSON_EXTENSION class.

Reply:
updated